src/pyinstaller_versionfile/__main__.py

```diff
@@ -18,8 +18,7 @@ def main(args=None):
 
 def parse_args(args):
     parser = argparse.ArgumentParser(description="Create a version file for PyInstaller from a YAML metadata file.")
-    parser.add_argument("metadata_file", help="Path to YAML metadata file")
-    parser.add_argument("--outfile", default="./version_file.txt", help="Resulting version file for PyInstaller")
+    parser.add_argument("--outfile", default="./file_version_info.txt", help="Resulting version file for PyInstaller")
```

Comment:
Two questions:

- is the removal of the positional argument (without replacement) intended?
- Why rename the default value for the output file? This would be a breaking change for users who rely on this default value, so if there is no urgent need for this, I would avoid changing it.

Reply:
line 21 happend by accident,
line 22 you're right, was a bad idea. file_version_info.txt is commonly used and also in readme, but not consequently. I'll rollback the commit